catt/controllers.py

```diff
@@ -54,15 +86,23 @@ def clear(self):
 
 
 class CastController:
-    def __init__(self):
+    def __init__(self, device_name):
         cache = Cache()
+        cached_ip = cache.get(device_name)
```

Comment:
Yes, but then you would only specify "living" or "bed" so you wouldn't have to get the entire name exactly right (case-sensitively, too).